### PR TITLE
Document build annotations for Koji task output

### PR DESCRIPTION
**Pull request by athos-ribeiro**
* OSBS-8138

Signed-off-by: Athos Ribeiro <athos@redhat.com>

### CODE REVIEW

File: docs/admins.rst [hunk 1]
```diff
@@ -367,6 +367,31 @@ Example:
...
+of successful container image builds. This file may include any wanted
+OpenShift build annotations for the container build triggered by the Koji task
+in question.
+
+The ``koji-containerbuild`` plugin looks for a
+``koji_task_annotations_whitelist`` annotation in the OpenShift build
+annotations. This key should hold a list of annotations to be whitelisted for
+inclusion in the ``build_annotations.json`` file.
+
+If an empty ``build_annotations.json`` file would be generated through the
```
Comment (rcerven) [inline docs/admins.rst]: If ``koji_task_annotations_whitelist`` is empty, ``build_annotations.json`` file is ommited from the task output

File: docs/admins.rst [hunk 2]
```diff
@@ -367,6 +367,31 @@ Example:
...
+process described above, the file is omitted from the task output.
+
+A list of annotations to be whitelisted in the
```
Comment (rcerven) [inline docs/admins.rst]: To whitelist openshift annotations described above, you have to configure it in ``reactor_config_map``  ``koji`` section ``task_annotations_whitelist``. See  :ref:`config.yaml` for further reference.

File: docs/admins.rst (hunk 1 above)
Comment (athos-ribeiro) [reply docs/admins.rst]: This is not the only possible cause for the file to be omitted though...
For instance, the whitelist could be populated with annotations not present in the OpenShift build annotations. 

I added a sentence with possible causes for the file being omitted.

File: docs/admins.rst (hunk 2 above)
Comment (athos-ribeiro) [reply docs/admins.rst]: I was trying to explain that the reactor config map option will set the annotation that whitelists other annotations to be included in the JSON file. I tried to re-word the sentence. WDYT?

File: docs/admins.rst (hunk 1 above)
Comment (rcerven) [reply docs/admins.rst]: ok

File: docs/admins.rst (hunk 2 above)
Comment (rcerven) [reply docs/admins.rst]: ok

**Pull request merged**